src/Compilers/CSharp/Portable/Symbols/ExtraAnnotations.cs

```diff
@@ -15,8 +15,9 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     internal enum AttributeAnnotations
     {
         None = 0,
-        NotNullWhenFalse,
-        EnsuresNotNull,
+        NotNullWhenTrue = 1 << 0,
+        NotNullWhenFalse = 1 << 1,
```

Comment:
Minor point: `= 1` and `= 2` seem easier to read.

Reply:
1 and 2 are indeed easier to read, but when the enum grows to 5 or 6 members, I believe this format will be easier to read.